src/v/cluster/members_manager.cc

```diff
@@ -278,6 +283,10 @@ members_manager::apply_update(model::record_batch b) {
       [this](register_node_uuid_cmd cmd) {
           const auto& node_uuid = cmd.key;
           const auto& requested_node_id = cmd.value;
+          vlog(
+            clusterlog.info,
+            "Applying register_node_uuid_cmd for UUID {}",
+            node_uuid);
```

Comment:
nit: this should be a part of a commit introducing handling for `register_node_uuid_cmd`